test/unit/client.test.ts

```diff
@@ -1113,4 +1117,45 @@ describe('Client', () => {
       expect(sdk.fetchUsersStation).not.toHaveBeenCalled();
     });
   });
+
+  describe('initialize', () => {
+    let orgSpy: jest.SpyInstance;
+    let userSpy: jest.SpyInstance;
+    let setupScSpy: jest.SpyInstance;
+    let proxySpy: jest.SpyInstance;
+    let requestSpy: jest.SpyInstance;
+
+    function setupSpys () {
+      orgSpy = jest.spyOn(sdk, 'fetchOrganization');
+      userSpy = jest.spyOn(sdk, 'fetchAuthenticatedUser');
+    }
+
+    beforeEach(() => {
+      setupScSpy = jest.spyOn(clientPrivate, 'setupStreamingClient').mockResolvedValue(null);
+      proxySpy = jest.spyOn(clientPrivate, 'proxyStreamingClientEvents').mockResolvedValue(null);
+      requestSpy = jest.spyOn(utils, 'requestApi');
+    });
+    
+    it('should init with jwt', async () => {
+      jwtDecodeSpy.mockReturnValue({
+        name: 'scooby',
+        org: 'myorg',
+        data: {
+          uid: 'myuserid',
+          jid: 'myjid',
+        }
+      } as any);
```

Comment:
Codacy has a fix for the issue: [Unexpected any. Specify a different type.](https://app.codacy.com/gh/MyPureCloud/genesys-cloud-webrtc-sdk/pullRequest?prid=9580329)
```suggestion
      } as unknown);
```